checkov/bicep/runner.py

```diff
@@ -108,6 +109,9 @@ def run(
 
         return report
 
+    def set_definitions_raw(self, definitions_raw: dict[Path, List[Tuple[int, str]]]) -> None:
```

Comment:
```suggestion
    def set_definitions_raw(self, definitions_raw: dict[Path, list[tuple[int, str]]]) -> None:
```
and please remove the added imports for `List, Tuple` you can use the built-in types as long as the file has
```python
from __future__ import annotations
```
at the beginning 🙂